.sync/Version.njk

```diff
@@ -36,4 +36,4 @@
 {% set latest_mu_release_branch = "release/202208" %}
 
 {# The version of the fedora-35-build container to use. #}
-{% set linux_build_container = "ghcr.io/tianocore/containers/fedora-35-build:5b8a008" %}
+{% set linux_build_container = "ghcr.io/tianocore/containers/fedora-37-build:14d2aba" %}

```

Comment:
Should we update the devcontainer too?

Reply:
Yes, I'll add that

Reply:
Updated in PR commit 6e7f3b7585975d9da393ffb6ba111e60b09948eb